Tribler/Core/leveldbstore.py

```diff
@@ -83,6 +82,14 @@ def __getitem__(self, key):
         try:
             return self._pending_torrents[key]
         except KeyError:
+            pass
+
+        try:
+            return self._db.Get(key)
+        except:
+            #self.flush()
```

Comment:
Let's try to sleep() for a bit? It's as ugly as it can get but let's see if it helps